### ISSUE TITLE
Use Material Design

### ISSUE THREAD

**Issue by sedrubal** [m1ga/pusteblume#24]
I think it's hard to develop and maintain a custom design, that is intuitive handy and pretty. Why don't you use a default android theme? I think the android 5 material design looks good and is quite handy. And I think that it is quite easy to make an option to select light or dark design (see #14) as many apps do this.

**Comment by m1ga** [comment]
Material is a really nice layout and an absolute must for the app. Actually a pull request was just merged that is adding "elevation" (https://github.com/appcelerator/titanium_mobile/pull/6931) so a nice card layout (http://www.google.com/design/spec/components/cards.html#cards-content) can be created!

**Comment by sedrubal** [comment]
Wow great :D

**Comment by manumaticx** [comment]
@m1ga These cards are implemented as [CardView](http://developer.android.com/training/material/lists-cards.html#CardView) which are not yet available in Titanium ([TIMOB-17361](https://jira.appcelerator.org/browse/TIMOB-17361))

**Comment by m1ga** [comment]
@manumaticx yeah, I saw the ticket. Hopefully soon :) But I think I could fake it with normal views (without the shadow)
